lib/errors.ts

```diff
@@ -156,6 +156,7 @@ const EXPECTED_ERROR_REGEXES = [
 	/must also be provided when using/, // oclif parser (depends-on)
 	/^Expected an integer/, // oclif parser (flags.integer)
 	/^Flag .* expects a value/, // oclif parser
+	/^Error parsing config file.*.balenarc.yml/,
```

Comment:
The two dots in the substring `.balenarc.yml` are interpreted in the regex sense of "any character" and I guess this was not intentional, although it works. Note that:
* The homedir configuration file is named `_balenarc.yml` on Windows and `.balenarc.yml` on Linux/macOS.
* A file named `balenarc.yml` (without a `_` or `.` as prefixes) on the current working directory takes precedence over the homedir configuration file ([documentation](https://github.com/balena-io/balena-cli-advanced-masterclass/blob/master/README.md#13-separate-environment-configurations)).

The regex works anyway because `.*.` means the same as `.*`, but I think it would be more... "canonical" to write:
```
/^Error parsing config file.*balenarc\.yml/,
```